lib/commands/command-list-all.bash

```diff
@@ -4,48 +4,42 @@ list_all_command() {
   local plugin_name=$1
   local query=$2
   local plugin_path
-  local std_out
-  local std_err
+  local std_out_file
+  local std_err_file
+  local output
   plugin_path=$(get_plugin_path "$plugin_name")
   check_if_plugin_exists "$plugin_name"
 
   # Capture return code to allow error handling
-  return_code=0 && split_outputs std_out std_err "bash ${plugin_path}/bin/list-all" || return_code=$?
```

Comment:
Rather than relying on this `split_outputs` function that keeps output in memory I opted to just write STDOUT and STDERR to separate temp files. This makes the code here much simpler. The downside is we are relying on the file system. But I figured this is fine because we already use temp files in several other places in the codebase.

Reply:
Fair enough choice :+1: